worldguard-legacy/build.gradle

```diff
@@ -2,6 +2,6 @@ dependencies {
     implementation 'org.spigotmc:spigot-api:1.12.2-R0.1-SNAPSHOT'
     implementation project(':core')
     // WG 6.2.2 was the last version before major API changes
-    implementation 'com.sk89q.worldguard:worldguard-legacy:6.2'
+    implementation 'com.sk89q.worldguard:worldguard-legacy:6.2.2'
```

Comment:
IIRC there was something weird about 6.2.2, and whatever it was seems to have caused the build to fail. I guess an additional comment should be added to mention that.

Reply:
It's weird, it's the last version before the major API changes.